AmazonChimeSDKDemo/AmazonChimeSDKDemo/AutomaticLanguageDetectionViewController.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  AutomaticLanguageDetectionViewController.swift
+//  AmazonChimeSDKDemo
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import AmazonChimeSDK
+import UIKit
+
+class AutomaticLanguageDetectionViewController: UIViewController, UITextFieldDelegate {
+    
+    @IBOutlet var languageDetectionTableView: UITableView!
+    @IBOutlet var preferredLanguagesTextField: UITextField!
+    var preferredLanguagesPickerView = UIPickerView()
```

Comment:
Can be singular? `preferredLanguagePickerView`, we are picking only a single preferred language value.

Reply:
will be updated in next commit